docs/how-to/notebooks.txt

```diff
@@ -0,0 +1,168 @@
+
+The ``--context`` option adds a folder or file to the notebook environment, allowing its contents to be accessed from within the notebook.
+
+.. code:: bash
+
+    det notebook start --context folder/file
+
+The ``--config-file`` option can be used to create a notebook with an environment specified by a configuration file.
+
+.. code:: bash
+
+    det notebook start --config-file config.yaml
+
+For more information on how to write the notebook configuration file, see :ref:`notebook-configuration`.
+
+Other Useful Commands
+#####################
+
+A full list of notebook commands can be found by running:
+
+.. code:: bash
+
+    det notebook --help
+
+To view all running notebooks:
+
+.. code:: bash
+
+    det notebook list
+
+To kill a notebook, you need its ID, which can be found using the list command.
+
+.. code:: bash
+
+    det notebook kill <id>
+
+WebUI
+~~~~~
+
+Notebooks can also be started from the WebUI by first going to the Determined home page. On the left-hand side, you can click the notebooks tab to take you to the notebooks page.
+
+.. image:: /assets/images/UI-Notebooks-View.png
+
+From here, you can find running notebooks. You can reopen, kill, or view logs for each notebook.
+
+To create a new notebook, click "Launch new notebook". If you would like to use a CPU-only notebook, click the dropdown arrow and select "Launch new CPU-only notebook".
+
+.. image:: /assets/images/UI-Notebooks-Dropdown.png
+
+.. _notebook-configuration:
+
+Notebook Configuration
+----------------------
+
+Notebooks may be supplied an optional **notebook configuration** to
+control aspects of the notebook's environment. For example, to launch a
+notebook that uses two GPUs:
+
+.. code::
+
+   $ det notebook start --config resources.slots=2
+
+In addition to the ``--config`` flag, configuration may also be supplied
+via a YAML file (``--config-file``):
+
+.. code::
+
+   $ cat > config.yaml <<EOL
+   description: test-notebook
+   resources:
+     slots: 2
+   bind_mounts:
+     - host_path: /data/notebook_scratch
+       container_path: /scratch
+   EOL
+   $ det notebook start --config-file config.yaml
+
+See :ref:`command-notebook-configuration` for more details on the supported
+configuration options.
+
+.. _cpu-only-notebooks:
+
+Example: CPU-Only Notebooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, each notebook is assigned a single GPU. This is appropriate for
+some uses of notebooks (e.g., training a deep learning model) but
+unnecessary for other tasks (e.g., analyzing the training
+metrics of a previously trained model). To launch a notebook that does
+not use any GPUs, set ``resources.slots`` to ``0``:
+
+.. code::
+
+   $ det notebook start --config resources.slots=0
+
+.. _notebook-state:
+
+Saving and Restoring Notebook State
+-----------------------------------
+
+.. warning::
+  It is only possible to save and restore notebook state on Determined clusters that
+  are configured with a shared filesystem available to all agents.
```

Comment:
should we move this warning up since it's a pretty non-obvious important point when working with notebooks currently? I know some users have ran into this problem a few times.

Reply:
Do you mean warning that users need a `shared_fs` to make notebooks persistent, or to warn that notebooks aren't persistent by default?

Reply:
More so notebooks aren't persistent by default.